docker-compose-mongo.yml

```diff
@@ -0,0 +1,53 @@
+version: '3'
+
+include:
+  - docker-compose.yml
+
+services:
+  mongo:

```

Comment:
Should we try to load in sample data for V2XGeoJson, CVCounts, and SSM collections? I think it could be nice for testing but we would probably want to enable/disable loading in sample data.

Reply:
Yeah sample data would be great.

Reply:
Pinging this again since it has been a while.

Reply:
Added an env variable and sample data script to insert into the V2XGeoJson, CVCounts, and OdeSsmJson collections